test/checks/aria/fallbackrole.js

```diff
@@ -13,28 +13,55 @@ describe('fallbackrole', function() {
       '<div id="target" role="button foobar">Foo</div>'
     );
     assert.isTrue(
-      checks.fallbackrole.evaluate(virtualNode.actualNode, 'radio', virtualNode)
+      checks.fallbackrole.evaluate(virtualNode.actualNode, null, virtualNode)
```

Comment:
the options param isn't actually used by the check, so it doesn't need to be in the test (unless there's a reason for it that I'm missing)